src/components/pages/Photos/roverNavbar/RoverNavbar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './RoverNavbar.css';
+
+export default function Navbar() {
+  return (
+    <ul className="rovernavbar">
+      <li className="roverbox">
```

Comment:
It would also be good for these list elements to be their own React Components - maybe `RoverButton`?